README.md

```diff
@@ -90,12 +90,12 @@ VCR follows the principles of [semantic versioning](https://semver.org/). The [A
 
 VCR versions 6.x are tested on the following ruby interpreters:
 
+  * MRI 3.1
   * MRI 3.0
   * MRI 2.7
   * MRI 2.6
-  * MRI 2.5
   
-Note that as of after VCR 6.0.0, only >= 2.5 is explicitly supported. 6.0.0 supported >= 2.4.
+Note that as of after VCR 6.0.0, only >= 2.6 is explicitly supported. 6.0.0 supports >= 2.4.
```

Comment:
6.0.0 was released. It supported 2.4.

Reply:
Perhaps we can formulate this in another way?

Reply:
True! I missed that. Sorry.

Just pushed another take trying to be more clear about the ruby support.